### DISCUSSION TITLE
How can I clean up test deployments in Workshop?

### DISCUSSION THREAD

**Discussion by dupuyarc** [noop-inc/documentation#125] [Questions] [answered]
I was able to deactivate the two deployments that I made (making the green button red), but this seems to just stop them (and maybe the VM?) from running, it doesn't offer to clean up databases or allocated storage for the containers etc.

Is there some way to completely remove something once I'm done with it?

**Comment by 1800joe** [comment, marked as answer]
I need a little more context to say for sure. Could you share a screenshot?

One possibility is to delete the environment, which is available from the general settings page. pressing the red "x" button will surface a confirmation to delete the Env

<img width="699" alt="Screenshot 2024-02-27 at 7 58 30 PM" src="https://github.com/noop-inc/documentation/assets/3111670/d23c3ed7-b98b-4034-a5d1-63f090036b72">

**Reply by dupuyarc** [reply]
OK, I missed that. I guess a red x-in-circle to me reads more about stopping something than deleting it. Maybe a trash can would be more intuitive for destroying the resources?

**Reply by dupuyarc** [reply]
Trying to find the menu you provided was rather hard, and I totally understand why I missed this. I had no idea that the "breadcrumb" boxes at the top were clickable, much less that they would display a drop-down menu, or that "Settings" and its screen (in the screenshot you sent me) would be the place where I would be able to delete a deployed environment.

<img width="1224" alt="Screenshot 2024-02-28 at 13 39 15" src="https://github.com/noop-inc/documentation/assets/157545233/ee5479e2-28f5-4bf8-abe4-d4b08ba01dee">

I created a separate enhancement suggestion / idea discussion #131 for some changes that would have helped me discover how to do this without needing to ask.
